AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -119,6 +121,8 @@ - (void)setUp {
   // Mock DistributeInfoTracker.
   self.distributeInfoTrackerMock = OCMClassMock([MSDistributeInfoTracker class]);
   self.sut.distributeInfoTracker = self.distributeInfoTrackerMock;
+    
```

Comment:
Remove this empty line or add a comment for the next line.
This is SDK coding convention, we don't do any line breaks unless there is a comment. In other words add a comment and explain what you are doing here since it starts a new code block for different things/behaviors than above.